platforms/database.go

```diff
@@ -5,6 +5,16 @@ import (
 	"strings"
 )
 
+// isKnownOS returns true if the operating system is Linux.
+//
+// The OS value should be normalized before calling this function.
+func isLinuxOS(os string) bool {
+	if os == "linux" {
+		return true
+	}
+	return false
```

Comment:
This if is unnecessary:

```go
return os == "linux"
```

not sure if a function is necessary for a single comparison.